### ISSUE TITLE
add data splitting code from test_split_data.ipynb in cheks_b2f to repo

### ISSUE THREAD

**Issue by donovanr** [AllenCellModeling/brightfield2fish#14]
* brightfield2fish version:
* Python version:
* Operating System:

### Description

Describe what you were trying to get done.
Tell us what happened, what went wrong, and what you expected to happen.

### What I Did

```
Paste the command(s) you ran and the output.
If there was a crash, please include the traceback here.
```

**Comment by donovanr** [comment]
addressed by 017f4efaa2f1f272450d22fc817813e3e25766ed

**Comment by donovanr** [comment]
whoops meant to close a different issue
